connection.js

```diff
@@ -255,44 +256,27 @@ class Connection {
         });
     }
     connection_reset() {
-        for (let game_id in this.connected_games) {
+        for (const game_id in this.connected_games) {
             this.disconnectFromGame(game_id);
         }
-        if (this.socket) this.socket.emit('notification/connect', this.auth({}), (x) => {
-            conn_log(x);
-        });
+        if (this.socket) {
+            this.socket.emit('notification/connect', this.auth({}), (x) => {
+            conn_log(x); });
```

Comment:
This is part of the previous line, should have different indentation.

Reply:
yes !